tasks/main.yml

```diff
@@ -40,6 +40,10 @@
   when: nginx_set_real_ip_from_cloudflare == True
   tags: [configuration, nginx]
 
+- include_tasks: amplify.yml
+  when: nginx_amplify == True
+  tags: [configuration, nginx, amplify]
```

Comment:
This apply "configuration" to all subtasks (there is package installation and some service launched inside the subtasks)

Reply:
Gotcha. This is a little bit tricky though since Nginx perform a number of configuration and service tasks in the `install.sh` script. I've added what I think are relevant tags to the individual steps inside `amplify.yml` but I would need some feedback as to whether this is correct :).